schemas/EiffelTestExecutionRecipeCollectionCreatedEvent/5.0.0.json

```diff
@@ -0,0 +1,295 @@
+{
```

Comment:
This should probably be 4.1.0 instead according to the versioning description here: https://github.com/eiffel-community/eiffel/blob/master/eiffel-syntax-and-usage/versioning.md